tests/Load/Dockerfile

```
@@ -1,18 +1,11 @@
-ENV CGO_ENABLED=0
-RUN go install go.k6.io/xk6/cmd/xk6@v0.11.0
+RUN go install go.k6.io/xk6/cmd/xk6@latest && \
+    xk6 build --with github.com/avitalique/xk6-file@latest \
+              --with github.com/mstoykov/xk6-counter@latest
 
-RUN xk6 build \
-    --with github.com/szkiba/xk6-faker@v0.3.0  \
-    --with github.com/mstoykov/xk6-counter@v0.0.1  \
-    --with github.com/grafana/xk6-exec@v0.3.0  \
-    --with github.com/avitalique/xk6-file@v1.4.0
+FROM grafana/k6:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)